var/spack/repos/builtin/packages/py-torch/package.py

```diff
@@ -100,6 +101,27 @@ class PyTorch(PythonPackage, CudaPackage):
     conflicts('+zstd', when='@:1.0')
     conflicts('+tbb', when='@:1.1')
 
+    cuda_arch_conflict = """This version of Torch/Caffe2 only supports compute
+    capabilities """
```

Comment:
This will cause newline characters and spaces to be in the message, I would use:
```python
cuda_arch_conflict = ('...'
    '...')
```
You can test it, but Python should automatically concatenate those strings. Alternatively, you can use:
```python
cuda_arch_conflict = '...' + \
    '...'
```
or something like that.

Reply:
It seemed to do the right thing in my testing.
```console
$ spack spec py-torch@1.2.0 cuda_arch=30
...
1. "cuda_arch=30" conflicts with "py-torch@1.2.0:+caffe2+cuda" [This version of Torch/Caffe2 only supports compute
    capabilities >=3.2]
```
but I can change if it desired.

Reply:
Oh, I see what you are saying.